### DISCUSSION TITLE
Read Smart objects in PSD Files

### DISCUSSION THREAD

**Discussion by ivanamar996** [dlemstra/Magick.NET#1331] [Help]
Hi,

Is it possible to read smart objects in PSD files and replace it with another PNG file?
Something like this:

![image](https://user-images.githubusercontent.com/39298540/211585815-504106d7-0ccf-4104-8104-0fd626028c4a.png)

![image](https://user-images.githubusercontent.com/39298540/211585881-9fcdb18a-d563-472c-8a29-b837023a39af.png)

Kind regards,
Ivana

**Comment by dlemstra** [comment]
The ImageMagick library does not support reading smart objects so you cannot do that with this library.
